app/controllers/officer/decisions_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Officer::DecisionsController < ApplicationController
+  def new
+    @application_decision = AssessmentDecisionForm.new(application_decision: nil)
+  end
+
+  def create
+    application = LandingApplication.find(params[:landing_application_id])
+    @application_decision = AssessmentDecisionForm.new(application_decision: params.dig(:assessment_decision_form, "application_decision"))
+
+    application.application_decision = @application_decision.application_decision
+
+    if @application_decision.valid? && application.valid?
```

Comment:
hmmm, it's been a long time since i looked at this, but I think we need to check `application.valid?`because that's where the validation for the enum value is. But maybe we don't need to check the application decision itself...